components/Tabs/tests/unit/MDCTabBarDisplayDelegateTests.m

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MDCItemBar.h"
+#import "MDCTabBarDisplayDelegate.h"
+#import "MaterialTabs.h"
+
+@interface MDCTabBarDisplayDelegate : NSObject <MDCTabBarDisplayDelegate>
+@property(nonatomic, assign) BOOL willDisplayItemWasCalled;
+@property(nonatomic, assign) BOOL didEndDisplayingItemWasCalled;
+@end
+
+@implementation MDCTabBarDisplayDelegate
+
+- (void)tabBar:(MDCTabBar *)tabBar willDisplayItem:(UITabBarItem *)item {
+  self.willDisplayItemWasCalled = YES;
+}
+
+- (void)tabBar:(MDCTabBar *)tabBar didEndDisplayingItem:(UITabBarItem *)item {
+  self.didEndDisplayingItemWasCalled = YES;
+}
+
+@end
+
+@interface MDCTabBarDisplayDelegateTests : XCTestCase
+@end
+
+@implementation MDCTabBarDisplayDelegateTests
+
+- (void)testMDCTabBarDisplayDelegateTabBarWillDisplayItem {
```

Comment:
Naming: Please name this to be include the condition that triggers the display path. In this case I'm guessing it's `layoutIfNeeded` or perhaps `setItems:`.